### PR TITLE
Fix/condo/doma 2503/fix add long string trimmer for table in property 2

### PR DESCRIPTION
**Pull request by VKislov**

### CODE REVIEW

File: apps/condo/domains/ticket/utils/clientSchema/Renders.tsx [hunk 1]
```diff
@@ -11,4 +11,14 @@ export const getTicketDetailsRender = (search: FilterValue) => {
 
         return getTableCellRenderer(search, false, null, null, null, details)(trimmedDetails)
     }
+}
+
+export const getTicketClientNameRender = (search: FilterValue) => {
+    return function render (ClientName: string, ticket: ITicketUIState) {
+        const address = get(ticket, ['property', 'address'])
+        const maxClientNameLength = address ? address.length : ClientName.length
```
Comment (akarjakin) [inline apps/condo/domains/ticket/utils/clientSchema/Renders.tsx:19]: all this calculations are not very clear. please add comments describing what and why is happening here.

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 2]
```diff
@@ -280,6 +280,7 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
                 render: renderUnit,
                 filterDropdown: getFilterDropdownByKey(filterMetas, 'unitName'),
                 filterIcon: getFilterIcon,
+                ellipsis:true,
```
Comment (sitozzz) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:283]:
```suggestion
                ellipsis: true,
```

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 3]
```diff
@@ -300,6 +301,7 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
                 filterDropdown: getFilterDropdownByKey(filterMetas, 'categoryClassifier'),
                 filterIcon: getFilterIcon,
                 render: renderClassifier,
+                ellipsis:true,
```
Comment (sitozzz) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:304]:
```suggestion
                ellipsis: true,
```

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 4]
```diff
@@ -310,8 +312,9 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
                 sorter: true,
                 width: columnWidths.clientName,
                 filterDropdown: getFilterDropdownByKey(filterMetas, 'clientName'),
-                render: getTableCellRenderer(search),
+                render: getTicketClientNameRender(search),
                 filterIcon: getFilterIcon,
+                ellipsis:true,
```
Comment (sitozzz) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:317]:
```suggestion
                ellipsis: true,
```

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 5]
```diff
@@ -324,6 +327,7 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
                 render: renderExecutor,
                 filterDropdown: getFilterDropdownByKey(filterMetas, 'executor'),
                 filterIcon: getFilterIcon,
+                ellipsis:true,
```
Comment (sitozzz) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:330]:
```suggestion
                ellipsis: true,
```

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 6]
```diff
@@ -336,6 +340,7 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
                 render: renderAssignee,
                 filterDropdown: getFilterDropdownByKey(filterMetas, 'assignee'),
                 filterIcon: getFilterIcon,
+                ellipsis:true,
```
Comment (sitozzz) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:343]:
```suggestion
                ellipsis: true,
```

File: apps/condo/domains/ticket/utils/clientSchema/Renders.tsx [hunk 7]
```diff
@@ -11,4 +11,15 @@ export const getTicketDetailsRender = (search: FilterValue) => {
...
+}
+
+// This function is needed to shorten the ClientName so that the field in which it is located is not unnecessarily stretched
+export const getTicketClientNameRender = (search: FilterValue) => { 
+    return function render (ClientName: string, ticket: ITicketUIState) {
+        const address = get(ticket, ['property', 'address'])
+        const maxClientNameLength = address ? address.length : ClientName.length
+        const trimmedClientName = get(ticket, ['ClientName'], '').length > maxClientNameLength ? `${ClientName.substring(0, maxClientNameLength)}…` : ClientName
+
+        return getTableCellRenderer(search, false, null, null, null, ClientName)(trimmedClientName)
```
Comment (pahaz) [inline apps/condo/domains/ticket/utils/clientSchema/Renders.tsx:23]: Looks like a magic

**Pull request merged**